### DISCUSSION TITLE
Native fullstack framework or libraries for Bun?

### DISCUSSION THREAD

**Discussion by rempas** [oven-sh/bun#7317] [Q&A]
Is there a native fullstack framework for Bun? In case the term "native" is not clear, let me make it more clear. With "native", I mean a framework that officially supports Bun and doesn't work through the Node API or any other layer of compatibility. The exception to that is when using the Node API for things that haven't implemented natively in Bun. Then, of course, I understand.

Alternatively, are there any native libraries that can be combined to create a modern, fully featured web app with Bun?

**Comment by truongan07** [comment]
> Is there a native fullstack framework for Bun?

Yes and you can't need NodeJS to run your project. However, we are waiting Bun.sh implements : 
- natively PostgreSQL [ important ] or MariaDB or MySQL
- natively Redis [nice to have]
- URL.createObjectURL() & URL.revokeObjectURL()
- CompressionStream & DecompressionStream
- console.table

Native fullstack framework:
- wbjs [frontend]
- wbServer [backend & native Bun]

wbServer, you can't need to use any library to make DOM from backend. wbServer supports 16 methods you do this.

**Reply by rempas** [reply]
Thank you for your reply! Until then, are there any frameworks made for Node (like Svelte, Solid, Qwik, etc.) that can work on bun without bugs?

**Comment by zackradisic** [comment]
Check out our full stack dev server documentation page! https://bun.sh/docs/bundler/fullstack

**Reply by ross-rosario** [reply]
> Check out our full stack dev server documentation page! https://bun.sh/docs/bundler/fullstack

Awesome guide! I wonder, how up-to-date is it?
